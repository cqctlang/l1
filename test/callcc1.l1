foo = lambda(){
    @local x;
    x = callcc(lambda(k){
      z = k;
      k(2);
      return 1;
    });
    print(x); 
};
foo();
