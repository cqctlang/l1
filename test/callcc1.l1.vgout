entry: var params, 1 locals, 0 temps
L4: 0 params, 2 locals, 0 temps
L10: 1 params, 0 locals, 0 temps
2
