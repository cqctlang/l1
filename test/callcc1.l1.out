callcc1.l1:2: warning: variable defined but not referenced: z
entry: var params, 1 locals, 0 temps
L4: 0 params, 2 locals, 0 temps
L10: 1 params, 0 locals, 0 temps
2
